src/plugins/orangehrmPerformancePlugin/Controller/EmployeeTrackerLogsController.php

```diff
@@ -38,9 +41,9 @@ public function preRender(Request $request): void
     {
         $id = $request->attributes->getInt('id');
         $component = new Component('employee-tracker-logs');
-        $tracker = $this->getPerformanceTrackerService()->getPerformanceTrackerDao()->getPerformanceTrack($id);
+        $tracker = $this->getPerformanceTrackerService()->getPerformanceTrackerDao()->getPerformanceTracker($id);
 
-        if (!is_null($tracker)) {
+        if (!is_null($tracker) && $this->isPerformanceTrackerAccessible($tracker)) {
```

Comment:
```suggestion
        if (!is_null($tracker)) {
```
for permission checking I think you can implement `OrangeHRM\Core\Authorization\Controller\CapableViewController`

e.g. https://github.com/orangehrm/orangehrm/blob/main/src/plugins/orangehrmPimPlugin/Controller/BaseViewEmployeeController.php#L92-L96

```php
    /**
     * @inheritDoc
     */
    public function isCapable(Request $request): bool
    {
        $id = $request->attributes->has('id');
        return $this->getUserRoleManager()->isEntityAccessible(PerformanceTracker::class, $id);
    }
```
:thinking:

Reply:
ah yes was not sure whether to go with this. I'll make the change

Reply:
added